src/components/tx/modals/NftTransferModal/SendNftForm.tsx

```diff
@@ -177,7 +170,7 @@ const SendNftForm = ({ params, onSubmit }: SendNftFormProps) => {
                     <MenuItem key={item.address + item.id} value={item.id}>
                       <NftMenuItem
                         image={item.imageUri || item.logoUri}
-                        name={item.name || item.tokenName}
+                        name={item.name || 'Unknown NFT'}
```

Comment:
Most of them don't have any name, only a tokenName. Please restore.
Also some might have a symbol but not a name, I would use that as a third fallback. Same for the collection name.

Reply:
The `name` refers to the collection. Do you not think they should differ slightly? (will commit after we're sure):

- collections: `item.name || item.tokenSymbol || 'Unknown collection'`
- NFTs: `item.tokenName || item.name || item.tokenSymbol || 'Unknown NFT'`

Reply:
AFAIU, `name` refers to the name of the unique item, while `tokenName` is the name of the collection (they use the word token to refer to the contract).

Reply:
And this `name` field is almost always empty. Only a few collections have named items.

Reply:
I've just looked over `safe-react` and it always uses `name` for both fields, falling back to an empty string. It would be nice to have a specific fallback though. Wdyt?

Reply:
As per our discussion, we've settled for the following:

- Collections: `item.tokenName || item.tokenSymbol || 'Unknown collection'`
- NFTs: `item.name || (item.tokenName || item.tokenSymbol || '') + item.id`